<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWindow</class>
 <widget class="QWidget" name="ConnectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection Settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Connections:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableConnections">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNewConn">
       <property name="text">
        <string>New Connection</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="btnBusActivate">
         <property name="text">
          <string>Activate Selected Connection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemoveBus">
         <property name="text">
          <string>Remove Selected Connection</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="btnActivateAll">
         <property name="text">
          <string>Activate All Connections</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeactivateAll">
         <property name="text">
          <string>Deactivate All Connections</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Connection Type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbGVRET">
          <property name="text">
           <string>GVRET Serial</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbKvaser">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Kvaser</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbSocketCAN">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>SocketCAN</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbPort"/>
     </item>
     <item>
      <widget class="QCheckBox" name="ckSingleWire">
       <property name="text">
        <string>Single Wire Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckListenOnly">
       <property name="text">
        <string>Listen Only Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>CAN Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbSpeed"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btnOK">
         <property name="text">
          <string>Create New Connection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRevert">
         <property name="text">
          <string>Revert</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
